### PR TITLE
Fix security guide capitalization errors

### PR DESCRIPTION
**Pull request by maclover7**
Would have submitted to docrails, but this guide was just changed today,
and docrails doesn't have the most updated version :grimacing:

cc @vipulnsward

[ci skip]

### CODE REVIEW

File: guides/source/security.md [hunk 1]
```diff
@@ -41,24 +41,24 @@ NOTE: _HTTP is a stateless protocol. Sessions make it stateful._
...
 ```
 
-### Session id
+### Session ID
 
-NOTE: _The session id is a 32-character random hex string._
+NOTE: _The session ID is a 32-character random hex string._
 
-The session id is generated using `SecureRandom.hex` which generates a random hex string using platform specific methods (such as openssl, /dev/urandom or win32) for generating cryptographically secure random numbers. Currently it is not feasible to brute-force Rails' session ids.
+The session ID is generated using `SecureRandom.hex` which generates a random hex string using platform specific methods (such as OpenSSL, /dev/urandom or win32) for generating cryptographically secure random numbers. Currently it is not feasible to brute-force Rails' session IDs.
```
Comment (vipulnsward) [inline guides/source/security.md]: Win32

File: guides/source/security.md [hunk 2]
```diff
@@ -196,11 +196,11 @@ This attack method works by including malicious code or a link in a page that ac
 
 ![](images/csrf.png)
 
-In the [session chapter](#sessions) you have learned that most Rails applications use cookie-based sessions. Either they store the session id in the cookie and have a server-side session hash, or the entire session hash is on the client-side. In either case the browser will automatically send along the cookie on every request to a domain, if it can find a cookie for that domain. The controversial point is that if the request comes from a site of a different domain, it will also send the cookie. Let's start with an example:
+In the [session chapter](#sessions) you have learned that most Rails applications use cookie-based sessions. Either they store the session ID in the cookie and have a server-side session hash, or the entire session hash is on the client-side. In either case the browser will automatically send along the cookie on every request to a domain, if it can find a cookie for that domain. The controversial point is that if the request comes from a site of a different domain, it will also send the cookie. Let's start with an example:
```
Comment (vipulnsward) [inline guides/source/security.md]: Any changes here?

Comment (maclover7) [inline guides/source/security.md]: in the second line of the paragraph, `Either they store the session id` --> `Either they store the session ID`

File: guides/source/security.md (hunk 1 above)
Comment (maclover7) [inline guides/source/security.md]: Didn't catch that one 👍

**Comment by maclover7** [comment]
👍 thanks for the review @vipulnsward!

**Pull request merged**